.circleci/kernel-module-build-failures-check/20-test-for-build-failures.sh

```diff
@@ -1,8 +1,10 @@
 #!/usr/bin/env bash
 set -eo pipefail
 
+FAILURES_DIR=${FAILURES_DIR:-/tmp/failures}
+
 shopt -s nullglob
-cd /tmp/failures
+cd "$FAILURES_DIR"
 failure_files=(*/*/*.log)
```

Comment:
Is there a difference in performance between this and using "find"? Perhaps find isn't installed. Probably not worth changing. I leave it up to you.

Reply:
I'm not sure I follow, this change is meant to allow the directory being scanned for failures when building drivers in OSCI. This is because the builders in OSCI directly dump logs to `/FAILURES` and I didn't want to duplicate the script for this small change. How would `find` help in this situation?